HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+ Library dependencies are to be mapped. 
+ Electronic components utilized will be logged in a bill of materials (BoM) and KiCad files for any custom PCBs will be archived. 
+ Printed mechanical parts will have files saved and machined components' sketches will be included. 
+ Expectations, hurdles and solutions are to be tabled in documents both before and after the flight.
+
+\subsubsection{Vehicle}
+\label{deliverables-vehicle}
+The desired product at the end of the development is a modular HAB vehicle capable of reliably, predictably, and safely carrying a payload compatible with a n*cubesat size. 
+The vehicle is defined as the mechanically sound HAB structure and common electronic/software components necessary for payload deployment. 
+Mechanical soundness includes, but is not limited to, protection from disintegration, temperature, winds, pressure, charge buildup and vibration. 
+To this end a power system, APRS transmitter, pressure/temperature/humidity sensor, payload arming/disarming mechanism and all facilitating processing and memory elements. 
+There must also be a data analytics system to record flight data and provide information leading to more advanced HABs and missions. 
+This must consist of sensors, processing and memory required for the task.
+It is expected that the payload will be modular, fitting both an x-u form factor and having a power supply of its own. 
+This is because optimization of a HAB vehicle capable of handling any modular payload results in selection for a power system dedicated solely to the HAB vehicle board and merely arming/disarming the payload electronics. 
+Past issues have included adhesives melting, batteries breaking apart and damage due to environmental factors. Each of these contingencies is to be specifically tested and solved for.
+
+\subsubsection{Software and Payload}
+\label{deliverables-payload}
+Python, instead of C/C++ allows for greater accessibility and the tradeoff in moving isn't worth the increased development costs. 
+Previous HAB teams have considered this question, and given the resources on the team, their findings are restated. OpenCV 3 will continue to be used as a major part.
+
+The primary payload utilized is the WUAP2 payload. 
+This is to build on the previous WUAP payload while adding a NDVI system.
+Use of NDVI is to be investigated on the ground and considerable attention paid to how NDVI results may be linked to visual spectrum analysis of vegetation. 
+While specific to the payload mission selected, exploring the linking of different realms of sensor information is a pursuit which will further aid SPEX. 
+An example could be the linking of IMU attitude data to images at the corresponding time, which in turn may allow for correction of apparent areas due to the angle of the camera with respect to the horizon. 
```

Comment:
Has this information already been stated elsewhere? Also is this a mission objective? Should it be moved to long-term vision or is it expected to be worked on within the scope of this project given enough manpower and expertise?

Reply:
The payload is part of the project.

Reply:
PLANTHAB is part of the mission.